Userland/Libraries/LibC/time.cpp

```diff
@@ -89,15 +89,37 @@ char* ctime_r(const time_t* t, char* buf)
     return asctime_r(localtime_r(t, &tm_buf), buf);
 }
 
-static const int __seconds_per_day = 60 * 60 * 24;
+static constexpr i64 __seconds_per_day = 60 * 60 * 24;
+static constexpr i64 __seconds_per_4_years = (3 * 365 + 366) * __seconds_per_day;
+static constexpr i64 __seconds_per_100_years = (76 * 365 + 24 * 366) * __seconds_per_day;
+static constexpr i64 __seconds_per_400_years = (303 * 365 + 97 * 366) * __seconds_per_day;
```

Comment:
I'm probably better off putting these constants in `AK/Time.h` to reduce duplication between that and `time.cpp`.

Reply:
We have a `AK/DateConstants.h` file too now, for what it's worth.